tests/runtime/ray/test_integration.py

```diff
@@ -546,81 +547,89 @@ def test_run_and_get_output(
     """
     # Given
     run_id = runtime.create_workflow_run(wf.model)
+    _wait_to_finish_wf(run_id, runtime)
 
     # When
-    wf_results = runtime.get_workflow_run_outputs(run_id)
+    wf_results = runtime.get_workflow_run_outputs_non_blocking(run_id)
     intermediate_outputs = runtime.get_available_outputs(run_id)
 
     # Then
     assert wf_results == expected_outputs
     assert intermediate_outputs == expected_intermediate
 
 
-# This test is slow to run locally because:
-# - It installs packages inside a separated venv.
+# These tests are slow to run locally because:
+# - It installs packages inside a separated venv. This includes cloning the driver
+#   process' venv.
 # - Installing packages takes some time in general.
 # - Installing packages on company machines is very slow because of the antivirus
 #   scanning.
 @pytest.mark.slow
 # Ray mishandles log file handlers and we get "_io.FileIO [closed]"
 # unraisable exceptions. Last tested with Ray 2.3.0.
 @pytest.mark.filterwarnings("ignore::pytest.PytestUnraisableExceptionWarning")
-def test_import_package_inside_ray(runtime: _dag.RayRuntime):
-    """
-    This test uses already generated workflow def from json file.
-    If necessary it can be recreated using ./data/python_package/original_workflow.py
-
-    This function tests 2 things
-    1. Proper package installation. Python Package defined in task definition
-    should only be installed inside venv of ray task and shouldn't leak into main env
-    2. We can create lib-dependent constant objects in the workflow definition and pass
-    them to task. This checks that we don't need python package outside an actual
-    task venv to deal with package-dependent constant values
-
-    We test those 2 things together as package installation is long, and we don't
-    want to do that twice unnecessarily
-    """
-    # This package should not be installed before running test
-    with pytest.raises(ModuleNotFoundError):
-        import polars  # type: ignore # noqa
+class Test3rdPartyLibraries:
+    @staticmethod
+    def test_constants_n_inline_imports(runtime: _dag.RayRuntime):
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Please update this test name, it looks incomplete.

Reply:
What name would make more sense to you?

Reply:
Reworded in 34627d5